.github/copilot-instructions.md

```diff
@@ -27,10 +27,14 @@
 * Use 'afterEach' for cleanup
 * Use 'expect' for assertions
 
+## Commits
+
+* Use conventional commits format: `<type>(<scope>): <subject>`
```

Comment:
Based on this documentation, the format seems to be little different
https://www.conventionalcommits.org/en/v1.0.0/

```
<type>[optional scope]: <description>

[optional body]

[optional footer(s)]
```

Reply:
body and footer are missing from the declaration, but otherwise it matches:

A scope may be provided to a commit’s type, to provide additional contextual information and is contained within parenthesis, e.g., feat(parser): add ability to parse arrays.

Reply:
But the scope should be optional.